installer/install.iss

```diff
@@ -2287,11 +2287,12 @@ begin
     // No credential helper
     RdbGitCredentialManager[GCM_None]:=CreateRadioButton(GitCredentialManagerPage,'None','Do not use a credential helper.',TabOrder,Top,Left);
 
-    // Restore the settings chosen during a previous install, if .NET 4.5.1
+    // Restore the settings chosen during a previous install, if .NET 4.7.2
     // or later is available.
-    if DetectNetFxVersion()<378675 then begin
+    if DetectNetFxVersion()<461808 then begin
         RdbGitCredentialManager[GCM].Checked:=False;
         RdbGitCredentialManager[GCM].Enabled:=False;
+        RdbGitCredentialManager[GCM].Caption:=RdbGitCredentialManager[GCM].Caption+' (requires .NET >= 4.7.2)'
```

Comment:
You'll want to specify .NET _Framework_ here rather than just ".NET" which is really the evolution of .NET Core after 3.x (Core being a separate product from Framework).

Otherwise someone may install ".NET 6" and wonder why it's still not working.

```suggestion
        RdbGitCredentialManager[GCM].Caption:=RdbGitCredentialManager[GCM].Caption+' (requires .NET Framework >= 4.7.2)'
```

Core CLR|Desktop CLR
-|-
.NET Core 1<br/>.NET Core 2.x<br/>.NET Core 3.x<br/>.NET 5<br/>.NET 6<br/>.NET 7|.NET Framework 1<br/>.NET Framework 2<br/>.NET Framework 3<br/>.NET Framework 3.5<br/>.NET Framework 4.0<br/>.NET Framework 4.5.x<br/>.NET Framework 4.6.x<br/>.NET Framework 4.7.x<br/>.NET Framework 4.8

GCM still needs the Desktop flavours of the CLR(!)

Reply:
Thanks, addressed in https://github.com/git-for-windows/build-extra/pull/329/commits/5db650c0aec90ecd7e3c193bec51a18ad5c5ed90 (I also touched up the code comment)